lib/modules/manager/asdf/upgradeable-tooling.ts

```diff
@@ -311,6 +311,15 @@ export const upgradeableTooling: Record<string, ToolingDefinition> = {
       extractVersion: '^php-(?<version>\\S+)',
     },
   },
+  pnpm: {
+    asdfPluginUrl: 'https://github.com/jonathanmorley/asdf-pnpm',
+    config: {
+      datasource: GithubReleasesDatasource.id,
```

Comment:
better use npm datasource, so we're sure the package was published successfully.

Reply:
it's just another import, not really more complex

Reply:
Which datasource would you use?